src/test/java/core/basesyntax/CalculatorTest.java

```diff
@@ -0,0 +1,106 @@
+package core.basesyntax;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class CalculatorTest {
+    private static Calculator calculator;
+
+    @BeforeEach
+    void init() {
+        calculator = new Calculator();
+    }
+
+    @Test
+    void addition_positive_ok() {
+        assertEquals(4, calculator.calculate(2, 2, '+'));
+        assertEquals(143234, calculator.calculate(72143, 71091, '+'));
+    }
+
+    @Test
+    void addition_negative_result_ok() {
+        assertEquals(-17, calculator.calculate(-9, -8, '+'));
+        assertEquals(-143234, calculator.calculate(-72143, -71091, '+'));
+    }
+
+    @Test
+    void addition_negative_ok() {
```

Comment:
```suggestion
    void addition_positive_negative_ok() {
```
these are just recommendations

same for other operations